elasticjob-lite/elasticjob-lite-lifecycle/src/main/java/org/apache/shardingsphere/elasticjob/lite/lifecycle/internal/settings/JobConfigurationAPIImpl.java

```diff
@@ -36,7 +36,11 @@
     
     @Override
     public JobConfigurationPOJO getJobConfiguration(final String jobName) {
-        return YamlEngine.unmarshal(regCenter.get(new JobNodePath(jobName).getConfigNodePath()), JobConfigurationPOJO.class);
+        String yamlContent = regCenter.get(new JobNodePath(jobName).getConfigNodePath());
```

Comment:
How about like this:
Optional.ofNullable(yamlContent).map(v -> YamlEngine.unmarshal(v, JobConfigurationPOJO.class)).orElse(null) ?
@TeslaCN @stanstanwang